tests/tests_subscriptions.py

```diff
@@ -128,7 +128,7 @@ def test_subscription_count_list(self):
             val = copy(val)  # we do not want to modify object in our db, we need a copy in order to generate event
             val.append(i)
             var.set_value(copy(val))
-        time.sleep(0.2)  # let last event arrive
```

Comment:
This was too short and causing the test to occasionally fail. I have increased it to 0.3.